pkg/api/handlers/utils/containers.go

```diff
@@ -238,3 +239,23 @@ func containerExists(ctx context.Context, name string) (bool, error) {
 	}
 	return ctrExistRep.Value, nil
 }
+
+// PSTitles merges CAPS headers from ps output. All PS headers are single words, except for
+// CAPS. Function compines CAP Headers into single field separated by a space.
+func PSTitles(output string) []string {
+	var titles []string
+
+	for _, title := range strings.Fields(output) {
+		switch title {
+		case "AMBIENT", "INHERITED", "PERMITTED", "EFFECTIVE", "BOUNDING":
+			{
+				titles = append(titles, title+" CAPS")
+			}
+		case "CAPS":
+			continue
+		default:
+			titles = append(titles, title)
+		}
```

Comment:
Can we somehow expose this in psgo? This looks rather hacky and will break every-time psgo adds a new header with space. cc @vrothberg

Reply:
This functionality doesn't look being tested.  A flip in `psgo` wouldn't hurt.

Reply:
Are we guaranteed to go through psgo all the time, don't we sometimes still exec ps?

While a new "space" added filed could cause an issue, it would be only a bad formatting on screen, not an actual bug.

Moving this function to psgo makes sense or having psgo expose all of these fields so we could get the list from it would make sense.